apps/datahub/src/app/home/navigation-menu/navigation-menu.component.ts

```diff
@@ -39,7 +39,7 @@ export class NavigationMenuComponent {
   activeLink$ = this.routerFacade.currentRoute$.pipe(
     map(
       (route) =>
-        this.tabLinks.find((tab) => tab.link === route.url[0].path) || {
+        this.tabLinks.find((tab) => tab.link === route.url[0]?.path) || {
```

Comment:
same ?

Reply:
With my fist attempt, when `/home` was hit, some errors were displayed in the console and a blank page appeared.
it doesn't happen with your imrovements. Thansk @fgravin !